README.md

```diff
@@ -1358,6 +1452,29 @@ Retrieves all the given object's own enumerable property values.
 
 **[⬆ back to top](#quick-links)**
 
+### _.get
+
+Gets the value at path of object.
+
+  ```js
+  // Lodash
+  var object = { a: [{ b: { c: 3 } }] };
+  var result = _.get(object, 'a[0].b.c', 1);
+  console.log(result);
+  // output: 3
+
+  // Native
```

Comment:
Need to consider that Lodash's _.get can have default values if the value doesn't exist.
Object destructure assumes the path is valid for an object.
like below example, if `c` doesn't exist, it will generate error using this approach.

Reply:
@cht8687 Yes, you are right but this is the only way to replace the lodash's `get` method with vanilla JS. But we can also put a note about that specific issue.

Otherwise the only "ugly" way we can state is:

```javascript
let a = object.a || {};
a.b= a.b || {};

let objValue = a.b.c || 'default';
```

Reply:
@zeeshanu agreed. Put a note about this issue is fine.